RFCs/FS-1071-witness-passing-quotations.md

```diff
@@ -300,10 +833,52 @@ bty.GetMethod("MyInlineFunction") // we want this to continue to succeed without
 # Unresolved questions
 [unresolved]: #unresolved-questions
 
-* [ ] The `CallWithWitnesses` node requires both the legacy and updated MethodInfos to be provided. This may be painful if people
-      attempt to create this node from nothing.
+## Witnesses for built in primitives
+[unresolved-builtin-witnesses]: #witnesses-for-built-in-primitives
+
+The list of new builtin withess solution primitives is long and bloats FSharp.Core. As an alternative it seems possible to provide one **generic** witness method for each `op_Addition` etc. and use the "type specialization" capability of .NET to ensure that efficient machine code is generated for each particular value-type instantiation.  Specifically .NET Core has a type-equality optimization for code generation that allows C# code like this:
```

Comment:
This seems like a much better route to me. Are there downsides?

Reply:
I don't think so.  Working through the details today/tomorrow.  I can likely unify with the existing `AdditionDynamic` and some other cases too so the API change looks like a completion rather than complexification

Reply:
Does this RFC need to be updated given the changes happening in the code?

Reply:
Yes